containerized.mk

```diff
@@ -39,19 +59,23 @@ ensure_sync_started:
 # to kill running containers (can't kill with ctrl-c)
 .PHONY: run
 run: ensure_image_exists
-	@ [ "$$DOCKER_SWARMKIT_DOCKER_RUN_CMD" ] || exit 1
-	@ DOCKER_RUN_COMMAND="docker run -t -v swarmkit-cache:${GOPATH}" \
+	[ "$$DOCKER_SWARMKIT_DOCKER_RUN_CMD" ] || exit 1
+	if [ "$$DOCKER_SWARMKIT_NOMOUNT" ]; then \
+			DOCKER_RUN_COMMAND="docker run -t"; \
+	else \
+		DOCKER_RUN_COMMAND="docker run -t -v swarmkit-cache:${GOPATH}" \
```

Comment:
I think you still want the `-v swarmkit-cache:${GOPATH}`: that's not a mount, it's just a docker volume (living only on the daemon's host) meant to cache go build artifacts to speed up builds on new containers.